src/hooks/init.js

```diff
@@ -19,9 +19,7 @@ module.exports = async function initHooks(context) {
     globalConfig.set('telemetryDisabled', false)
     console.log('Netlify telemetry has been enabled')
     console.log('You can disable it anytime with the --telemetry-disable flag')
-    track('user_telemetryEnabled', {
```

Comment:
This was a bug since `track` is `async` (because it reads the global config). We need to `await` it otherwise `process.exit()` is called before starting the child process that sends the telemetry request.